src/Core/Models/Api/Request/Accounts/VerifyOtpRequestModel.cs

```diff
@@ -2,9 +2,9 @@
 
 namespace Bit.Core.Models.Api
 {
-    public class ApiKeyRequestModel
+    public class VerifyOtpRequestModel
```

Comment:
> Re: Otp -> OTP 

_(we lost the original thread it seems)_

We do hacks converting between camel and snake case that effect the 3'rd+ capitals of object names? Wouldn't any hacks just apply to first capitals?

Reply:
We do those hacks on responses, so request should be safe. I've renamed the class as well (I only renamed the field before)